procrastinate/jobs.py

```diff
@@ -47,6 +47,8 @@ class Job:
     ----------
     id :
         Internal id uniquely identifying the job.
+    status :
+        Status of the job.
```

Comment:
Oh, I didn't excpet we'd finally add that. I don't remember why we didn't model this at first. I'd say, because this is obvious when we defer jobs, and obvious when the worker treats them, so there was no use. Now with the shell and all the ways to get jobs in various states, it makes sense.